var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -29,19 +29,19 @@
 from spack import *
 
 
-class Rose(Package):
+class Rose(AutotoolsPackage):
     """A compiler infrastructure to build source-to-source program
        transformation and analysis tools.
        (Developed at Lawrence Livermore National Lab)"""
 
     homepage = "http://rosecompiler.org/"
-    url = "https://github.com/rose-compiler/rose/archive/v0.9.7.tar.gz"
+    #url = "https://github.com/rose-compiler/rose/archive/v0.9.7.tar.gz"
```

Comment:
Yeah......... quite a shame, I agree.

I'm trying to use Git, but It doesn't seem to work when I comment out the URL. The tarball is still being downloaded... Is there something in a cache that I need to clear? How can I run `spack install rose@0.9.9.0` using the Git repository as specified?